lib/nimble_options/docs.ex

```diff
@@ -84,25 +77,25 @@ defmodule NimbleOptions.Docs do
   end
 
   defp get_required_str(schema) do
-    if schema[:required], do: "required ", else: ""
+    if schema[:required], do: "Required."
   end
 
   defp get_default_str(schema) do
-    if Keyword.has_key?(schema, :default) do
-      "The default value is `#{inspect(schema[:default])}`."
-    end
+    if Keyword.has_key?(schema, :default),
+      do: "The default value is `#{inspect(schema[:default])}`."
   end
 
   defp get_type_str(schema) do
     case schema[:type] do
-      {:custom, _module, _function, _args} -> "`custom`"
-      {:fun, arity} -> "`function/#{arity}`"
-      {:keyword_list, _} -> "`keyword`"
-      {:in, values} -> "one of `" <> Enum.map_join(values, "`, `", &get_type_str(type: &1)) <> "`"
-      {:list, subtype} -> "`list(#{subtype})`"
-      {:non_empty_keyword_list, _} -> "`keyword`"
+      nil -> nil
+      {:custom, _module, _function, _args} -> nil
+      {:fun, arity} -> "function/#{arity}"
+      {:keyword_list, _} -> "keyword"
+      {:in, values} -> "one of " <> Enum.map_join(values, ", ", &get_type_str(type: &1))
+      {:list, subtype} -> "list(#{subtype})"
+      {:non_empty_keyword_list, _} -> "keyword"
       {:or, values} -> Enum.map_join(values, " or ", &get_type_str(type: &1))
-      _type -> "`" <> String.trim(to_string(schema[:type])) <> "`"
+      _type -> String.trim(to_string(schema[:type]))
```

Comment:
These are the basic types?

Reply:
Yes this is the fallback for basic types we can format normally.